### PR TITLE
added FastExp

### PR DESCRIPTION
**Pull request by blockparole**
Added fastexp with autoswitch and autothrow settings.

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+/**
+ * Created 17 October 2019 by hub
+ * Updated 21 November 2019 by hub
+ */
+@Module.Info(name = "FastExp", category = Module.Category.PLAYER, description = "Auto Switch to XP and throw fast")
+public class FastExp extends Module {
+
+    private Setting<Boolean> autoThrow = register(Settings.b("Auto Throw", true));
+    private Setting<SwitchMode> switchMode = register(Settings.e("Auto Switch", SwitchMode.ON));
```
Comment (zeroeightysix) [inline src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java]: Why not a boolean setting?

File: src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                mc.player.inventory.currentItem = initHotbarSlot;
+            }
+        }
+
+    }
+
+    @Override
+    public void onUpdate() {
+
+        if (isDisabled() || mc.player == null) {
```
Comment (zeroeightysix) [inline src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java]: `onUpdate()` will never be called when the module is disabled (unless `alwaysListening` is true for the module, iirc)
No need to check for `isDisabled()`!

Comment (blockparole) [reply src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java]: Resolved [here](https://github.com/zeroeightysix/KAMI/pull/144/commits/5f41766ef2aab8db2c5f2f3526d3bda5cb839d8a).

File: src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java (hunk 1 above)
Comment (blockparole) [reply src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java]: changed to `Setting<Boolean>` [here](https://github.com/zeroeightysix/KAMI/pull/144/commits/c57ea9eddf376f28ef179b71be984c2da6768aa6).

**Comment by zeroeightysix** [comment]
Maybe make the player look down (with a setting) while throwing xp bottles?

**Comment by S-B99** [comment]
Spoofing this (as an option) would be good as well

**Pull request merged**